### PR TITLE
update bvt test for KVVM Ext

### PR DESCRIPTION
**Pull request by lanbofang**
Update BVT test for extension 3.x

### CODE REVIEW

**Comment by lanbofang** [comment]
@microsoft-github-policy-service agree


________________________________

File: microsoft/testsuites/vm_extensions/azure_keyvault_vm_extension.py [hunk 1]
```diff
@@ -51,10 +52,29 @@ def _check_system_status(node: Node, log: Logger) -> None:
...
-        "/var/lib/waagent/Microsoft.Azure.KeyVault -la", sudo=True
+        "/var/lib/waagent -la", sudo=True
     ).stdout
     log.info(f"Directory contents: {directory_contents}")
 
+    # check certs files
+    certa = "/var/lib/waagent/a/symbolinka"
+    message = f"File {certa} was not created on the test machine"
+    node.execute(
+        f"ls '{certa}'",
```
Comment (LiliDeng) [inline microsoft/testsuites/vm_extensions/azure_keyvault_vm_extension.py]: use ls tool `ls = node.tools[Ls]`

File: microsoft/testsuites/vm_extensions/azure_keyvault_vm_extension.py [hunk 2]
```diff
@@ -51,10 +52,29 @@ def _check_system_status(node: Node, log: Logger) -> None:
...
+        shell=True,
+        expected_exit_code=0,
+        expected_exit_code_failure_message=message,
+    )
+
+    certb = "/var/lib/waagent/a/symbolinkb"
+    message = f"File {certb} was not created on the test machine"
+    node.execute(
```
Comment (LiliDeng) [inline microsoft/testsuites/vm_extensions/azure_keyvault_vm_extension.py]: use `ls = node.tools[Ls]`

File: microsoft/testsuites/vm_extensions/azure_keyvault_vm_extension.py [hunk 3]
```diff
@@ -175,20 +194,40 @@ def verify_key_vault_extension(
...
-                    certificates_secret_id[0],
-                    certificates_secret_id[1],
-                ],
+                    {
+                        "url": certificates_secret_id[0],
+                        "certificateStoreLocation": "/var/lib/waagent/a",
+                        "customSymbolicLinkName": "symbolinka",
+                        "acls": [
+                            {
+                                "user": f"{current_user}"
```
Comment (LiliDeng) [inline microsoft/testsuites/vm_extensions/azure_keyvault_vm_extension.py]: "user": current_user

File: microsoft/testsuites/vm_extensions/azure_keyvault_vm_extension.py [hunk 4]
```diff
@@ -175,20 +194,40 @@ def verify_key_vault_extension(
...
+                            }
+                        ]
+                    },
+                    {
+                        "url": certificates_secret_id[1],
+                        "certificateStoreLocation": "/var/lib/waagent/b",
+                        "customSymbolicLinkName": "symbolinkb",
+                        "acls": [
+                            {
+                                "user": f"{current_user}"
```
Comment (LiliDeng) [inline microsoft/testsuites/vm_extensions/azure_keyvault_vm_extension.py]: ditto

File: microsoft/testsuites/vm_extensions/azure_keyvault_vm_extension.py (hunk 1 above)
Comment (lanbofang) [reply microsoft/testsuites/vm_extensions/azure_keyvault_vm_extension.py]: Updated.

File: microsoft/testsuites/vm_extensions/azure_keyvault_vm_extension.py (hunk 2 above)
Comment (lanbofang) [reply microsoft/testsuites/vm_extensions/azure_keyvault_vm_extension.py]: Updated.

**Review by LiliDeng — APPROVED** [review]

**Pull request merged**